save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt

```diff
@@ -189,9 +192,12 @@ fun topBar() = fc<TopBarProps> { props ->
                 // Dropdown - User Information
                 div("dropdown-menu dropdown-menu-right shadow animated--grow-in") {
                     attrs["aria-labelledby"] = "userDropdown"
-                    // FixMe: temporary disable Profile DropDown, will need to link it with the user in the future
-                    // dropdownEntry("cogs", "Profile")
-                    dropdownEntry("sign-out-alt", "Log out") {
+                    dropdownEntry(faCog, "Settings") {
+                        attrs.onClickFunction = {
+                            userSettings(props.userInfo?.name ?: "Unknown")
```

Comment:
I think we shouldn't display this dropdown for anonymous users at all and have a `log in` button instead. And now it should have not `Unknown`, but `Not logged in` or `Unauthenticated`

Reply:
I agree. But it's easier to test locally

Reply:
Yeah, but maybe at least we can change text now? Not `Unknown`, but `Unauthenticated`

Reply:
Ok. Added